_overviews/scala3-book/concurrency.md

```diff
@@ -180,38 +181,39 @@ See the [Futures and Promises][futures] page for a discussion of additional meth
 
 ## Running multiple futures and joining their results
 
-To run multiple futures in parallel and join their results when all of the futures complete, use a `for` expression.
+To run multiple computations in parallel and join their results when all of the futures have been completed, use a `for` expression.
+
 The correct approach is:
 
-1. Create the futures
+1. Run the computations that return `Future` results
 2. Merge their results in a `for` expression
 3. Extract the merged result using `onComplete` or a similar technique
 
 
 ### An example
 
 The three steps of the correct approach are shown in the following example.
-A key is that you first create the futures and then join them in the `for` expression:
+A key is that you first run the computations that return futures, and then join them in the `for` expression:
```

Comment:
You could maybe say “you first start the computations” instead of “you first run the computations”, but I think either is fine, depending on your preference.

Reply:
Changed for “start”